dali/operators/reader/loader/loader.h

```diff
@@ -70,7 +70,8 @@ class Loader {
       loading_flag_(false),
       read_sample_counter_(0),
       returned_sample_counter_(0),
-      pad_last_batch_(options.GetArgument<bool>("pad_last_batch")) {
+      pad_last_batch_(options.GetArgument<bool>("pad_last_batch")),
+      slab_anchor_({}), slab_shape_({}) {
```

Comment:
nitpick: Is there a special reason why this is called `slab` and not `slice`? I'd prefer the later, since it is extensively used in other parts of DALI

Reply:
that is totally fine, I agree. I wasn't consistent in what I want to call it. I renamed it to slice throughout. Will be fixed in the next commit. Thank you.